modules/external-api/src/main/java/org/opencastproject/external/endpoint/SeriesEndpoint.java

```diff
@@ -344,15 +359,45 @@ public JValue apply(SearchResultItem<Series> a) {
     }
   }
 
+  /**
+   * Get an {@link AccessControlList} from a {@link Series}.
+   *
+   * @param series
+   *          The {@link Series} to get the ACL from.
+   * @return The {@link AccessControlList} stored in the {@link Series}
+   */
+  private static AccessControlList getAclFromSeries(Series series) {
+    AccessControlList activeAcl = new AccessControlList();
+    try {
+      if (series.getAccessPolicy() != null)
+        activeAcl = AccessControlParser.parseAcl(series.getAccessPolicy());
```

Comment:
I remember a mailing list discussion about brace style and I *think* the vast majority of responses were in favor of Greg's proposal to make braces mandatory.

```suggestion
      if (series.getAccessPolicy() != null) {
        activeAcl = AccessControlParser.parseAcl(series.getAccessPolicy());
      }
```

Reply:
You're right. I simply copied this over from the EventsEndpoint, but I'll add some braces (I like braces!)